plugins/connection/aws_ssm.py

```diff
@@ -540,35 +541,34 @@ def _initialize_ssm_client(self, region_name: Optional[str], profile_name: str)
             None
         """
 
-        self._vvvv("SETUP BOTO3 CLIENTS: SSM")
+        self.verbosity_display(4, "SETUP BOTO3 CLIENTS: SSM")
         self._client = self._get_boto_client(
             "ssm",
             region_name=region_name,
             profile_name=profile_name,
         )
 
-    def _display(self, f: Any, message: str) -> None:
+    def verbosity_display(self, level: int, message: str) -> None:
+        """
+        Displays the given message depending on the verbosity level.
+
+        :param message: The message to display.
+        :param display_level: The verbosity level (1-4, default value of 1).
+
+        :return: None
+        """
         if self.host:
             host_args = {"host": self.host}
         else:
             host_args = {}
-        f(to_text(message), **host_args)
-
-    def _v(self, message: str) -> None:
-        self._display(display.v, message)
-
-    def _vv(self, message: str) -> None:
-        self._display(display.vv, message)
 
-    def _vvv(self, message: str) -> None:
-        self._display(display.vvv, message)
+        verbosity_level = {1: display.v, 2: display.vv, 3: display.vvv, 4: display.vvvv}
 
-    def _vvvv(self, message: str) -> None:
-        self._display(display.vvvv, message)
+        verbosity_level[level](to_text(message), **host_args)
```

Comment:
```suggestion
        if level not in verbosity_level.keys():
           level = 4
        verbosity_level[level](to_text(message), **host_args)
```

Either default to the highest level or raise an exception if the level is not part of the expected list

Reply:
Good call!